Packs/EmailCommunication/Scripts/PreprocessEmail/PreprocessEmail.py

```diff
@@ -0,0 +1,109 @@
+from CommonServerPython import *
+import sys
+import json
+
+
+def add_entries(single_reply, email_related_incident):
+    """Add the entries to the related incident
+    Args:
+        single_reply: The email reply.
+        email_related_incident: The related incident.
+    Returns:
+
+    """
+    entries_str = json.dumps([{"Type": 1, "ContentsFormat": 'markdown', "Contents": single_reply, "tags": ['email'
+                                                                                                           '-thread']}])
+    res = demisto.executeCommand("addEntries", {"entries": entries_str, 'id': email_related_incident})
+    if is_error(res):
+        demisto.results('ERROR: PreprocessEmail - addEntries: {}'.format(res['Contents']))
+        # Fail safe to not drop emails if we couldn't add entries to existing incident
+        demisto.results(True)
+        sys.exit(0)
```

Comment:
I think it would be better to return the value you desire, and not use `sys.exit` like that